drivers/util.py

```diff
@@ -1061,6 +1067,12 @@ def test_scsiserial(session, device):
                    % device)
         return False
 
+    # USB devices can have identical SCSI IDs - prefer matching with serial number
+    try:
+        usb_device_with_serial = serial and is_usb_device(device)
+    except CommandException:
+        usb_device_with_serial = False
```

Comment:
```suggestion
    except Exception as e:
        usb_device_with_serial = False
        SMlog(f"Unable to check if device is USB: {e}")
```

Reply:
I'm not a big fan of broad exception catch but ok. Regarding the log, just using f-string interpolation on exception is often not very helpful. Would it be ok for you to do the log like this:
```suggestion
    except:
        usb_device_with_serial = False
        SMlog("Unable to check if device is USB:")
        SMlog(traceback.format_exc())
```

Reply:
You can commit your suggestion, not a blocking point I guess.

Reply:
ok it is committed.